src/main/java/org/jabref/Launcher.java

```diff
@@ -148,7 +149,7 @@ private static void addLogToDisk() {
     }
 
     private static void initializeLogger() {
-        LOGGER = LoggerFactory.getLogger(MainApplication.class);
+        LOGGER = LoggerFactory.getLogger(JabRefGUI.class);
```

Comment:
Oh, Logger for `Launcher` called `JabRefGUI`. Really?

Reply:
Yes, seemed suspicious to mee to, but I just adapted it to the new architecture. Will change to `Launcher.class`.